### PR TITLE
build(circle): make simpler

### PR DESCRIPTION
**Pull request by jkurz**
* make the build great again

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -5,40 +5,35 @@ machine:
...
 dependencies:
   pre:
     - npm install
     - npm test
   override:
     - docker version
     - docker info
-    - docker login -e="${DOCKER_EMAIL}" -u="${DOCKER_USER}" -p="${DOCKER_PASSWORD}" quay.io
-    - docker run -v $(pwd):/data quay.io/turner/harbor-deploy set_build_values \
+    - docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USER" -p="$DOCKER_PASSWORD" quay.io
```
Comment (wilsonodk) [inline circle.yml]: Not sure why we are making these less Bash-y.

File: circle.yml [hunk 2]
```diff
@@ -5,40 +5,35 @@ machine:
...
+    - docker run -v `pwd`:/data quay.io/turner/harbor-deploy:0.2.0-add-catalog-function.16 set_build_values \
```
Comment (wilsonodk) [inline circle.yml]: Why would we revert to a deprecated syntax?

File: circle.yml [hunk 3]
```diff
@@ -5,40 +5,35 @@ machine:
...
-    - source buildprops; docker build -t "quay.io/turner/${HARBOR_CONTAINER}:${HARBOR_VERSION}" .
-    - source buildprops; docker push "quay.io/turner/${HARBOR_CONTAINER}:${HARBOR_VERSION}"
-    - source buildprops; docker run quay.io/turner/harbor-deploy deploy \
-      -s "ops-${HARBOR_CONTAINER}" \
-      -r quay.io/turner \
-      -c "${HARBOR_CONTAINER}" \
-      -v "${HARBOR_VERSION}" \
+      -b $CIRCLE_BRANCH \
+      -n $CIRCLE_BUILD_NUM > props;
+    - source props; docker build -t $IMAGE .
```
Comment (wilsonodk) [inline circle.yml]: I like this, but why aren't we doing `source props; docker build -t "${IMAGE}" . `

File: circle.yml [hunk 4]
```diff
@@ -5,40 +5,35 @@ machine:
     - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
   services:
     - docker
-
-test:
```
Comment (wilsonodk) [inline circle.yml]: Why are we removing this?

File: circle.yml [hunk 5]
```diff
@@ -20,25 +20,26 @@ dependencies:
...
     - docker login -e="${DOCKER_EMAIL}" -u="${DOCKER_USER}" -p="${DOCKER_PASSWORD}" quay.io
-    - docker run -v $(pwd):/data quay.io/turner/harbor-deploy set_build_values \
+    - docker run -v $(pwd):/data quay.io/turner/harbor-deploy:0.2.0-add-catalog-function.18 set_build_values \
       -d /data \
-      -b "${CIRCLE_BRANCH}" \
-      -n "${CIRCLE_BUILD_NUM}" > buildprops;
-    - source buildprops; docker build -t "quay.io/turner/${HARBOR_CONTAINER}:${HARBOR_VERSION}" .
-    - source buildprops; docker push "quay.io/turner/${HARBOR_CONTAINER}:${HARBOR_VERSION}"
-    - source buildprops; docker run quay.io/turner/harbor-deploy deploy \
+      -b ${CIRCLE_BRANCH} \
```
Comment (wilsonodk) [inline circle.yml]: We still want all these Bash variables wrapped in the double quotes.

```yml
dependencies:
  pre:
    - npm install
    - npm test
  override:
    - docker version
    - docker info
    - docker login -e="${DOCKER_EMAIL}" -u="${DOCKER_USER}" -p="${DOCKER_PASSWORD}" quay.io
    - docker run -v $(pwd):/data quay.io/turner/harbor-deploy:0.2.0-add-catalog-function.18 set_build_values \
      -d /data \
      -b "${CIRCLE_BRANCH}" \
      -n "${CIRCLE_BUILD_NUM}" > props;
    - source props; docker build -t "${IMAGE}" .
    - source props; docker push "${IMAGE}"
    - source props; docker run --env-file props quay.io/turner/harbor-deploy:0.2.0-add-catalog-function.18 catalog \
      -s "ops-${HARBOR_CONTAINER}" \
      -e dev \
      -t "${SHIPMENT_BUILD_TOKEN}"
```

**Comment by jkurz** [comment]
@wilsonodk making you happy is pretty much my job description.

**Pull request merged**